README.md

```diff
@@ -190,7 +190,7 @@ Contributions, issues, and feature requests are welcome!
 
```

Comment:
- [ ] Please make sure that the background image matches the design.
- [ ] please remove the weird black line that appears on the headline section icons.
- [ ] Please make sure that the distance on the left of the headline section icons matches the design.
- [ ] Please make sure that the order of the icons in the headline section matches the design
- [ ] Please make sure to remove the decorations on the `let's connect` button it doesn't match the design.

Your code 
![image](https://user-images.githubusercontent.com/74744787/227737577-33244eaf-f4ff-4998-9980-0982eaf453ce.png)
Figma design 
![image](https://user-images.githubusercontent.com/74744787/227737714-a6c91502-492b-4805-916c-6674650398a5.png)